--- a/etc/UPower.conf	2026-07-21 20:56:01.418031866 +0800
+++ b/etc/UPower.conf	2026-07-21 20:56:01.423031961 +0800
@@ -64,7 +64,7 @@
 # PercentageAction=2.0
 PercentageLow=20.0
 PercentageCritical=5.0
-PercentageAction=2.0
+PercentageAction=0
 
 # When UsePercentageForPolicy is false, the time remaining in seconds at
 # which UPower will consider the battery low, critical, or take action for
@@ -79,7 +79,7 @@
 # TimeAction=120
 TimeLow=1200
 TimeCritical=300
-TimeAction=120
+TimeAction=10
 
 # Enable the risky CriticalPowerAction-Suspend
 # This option is not recommended, but it is here for users who
